benchmark/kvbench/runtime/etcd_rt.py

```
@@ -140,14 +159,26 @@ def barrier(self, ranks: Optional[List[int]] = None, timeout_sec=600):
                 if timeout_sec and time.time() - start_time > timeout_sec:
+                    current_val = self._get_int_val(key) or 0
+                    waiting_for_rank = (
+                        ranks[my_index - 1] if my_index > 0 else "unknown"
+                    )
                     raise TimeoutError(
-                        f"[Rank {self.rank}] Barrier {key} timed out after {timeout_sec} seconds, current value: {self.client.get(key)}, waiting for val={my_index} (i.e rank {ranks[my_index - 1]} entered barrier), (ranks: {ranks})"
+                        f"[Rank {self.rank}] Barrier timed out after {timeout_sec:.0f}s: {current_val}/{len(ranks)} ranks arrived. "
+                        f"Waiting for rank {waiting_for_rank} to enter barrier."
                     )
             # Fan out - wait for root to set 0 again
             while not self._get_int_val(key) == 0:
```

Ruff (0.15.14)

[warning] Use `self._get_int_val(key) != 0` instead of `not self._get_int_val(key) == 0`

Replace with `!=` operator

(SIM201)